lineage/tHMM.py

```diff
@@ -52,9 +56,24 @@ def __init__(self, X, num_states, fpi=None, fT=None, fE=None):
         self.fpi = fpi
         self.fT = fT
         self.fE = fE
-        self.X = X  # list containing lineages, should be in correct format (contain no NaNs)
-        self.num_states = num_states  # number of discrete hidden states
+        self.X = X  # list containing lineages
+        self.num_states = num_states  # number of discrete hidden states, should be integral
         self.estimate = estimate(self.X, self.num_states, fpi=self.fpi, fT=self.fT, fE=self.fE)
+        self.EL = get_Emission_Likelihoods(self)
+
+    # getter method
+    def get_EL(self):
```

Comment:
What's the benefit of this over direct access?

Reply:
Nothing, I thought it'd be the right way to do it, but you're right, I'd rather prefer the cleaner/shorter way to do it.